ssh/main.go

```diff
@@ -23,20 +23,7 @@ const (
 )
 
 func init() {
-	level := log.InfoLevel
```

Comment:
Please split this rework in a specific commit. Then add a new commit for the new users of package.